docs/extend/plugins_volume.md

```diff
@@ -140,15 +162,15 @@ at the first mount request and deprovision at the last corresponding unmount req
 **Response**:
 ```json
 {
-    "Mountpoint": "/path/to/directory/on/host",
+    "Mountpoint": "/path/to/directory/inside/plugin",
```

Comment:
I don't have the knowledge about v1 and v2 that I obviously need to complete this task. Where can I find out about this?